Project.toml

```diff
@@ -4,7 +4,6 @@ authors = ["The developers of COBREXA.jl"]
 version = "1.1"
```

Comment:
I think you have to bump the version here

Reply:
Ah I didn't want to version this but we can do, OK

Reply:
I think it is always the idea when merging `develop` ->`master` to then tag the repo (even patch)

Reply:
ok